### ISSUE TITLE
Linux support?

### ISSUE THREAD

**Issue by Breelandthecat** [geode-sdk/geode#553]
### Geode Version

v2.0.0-Beta.20

### Related Problem

Can you please add official linux support?

I Recently went back to linux mint. I Moved from windows to linux, To windows again,  then back. I Cant install geode though. I Tried installing it through proton in the same version, But it didnt work. The geode files exist and stuff, but geometry dash acts as if geode doesnt exist or anything.

### Your Solution

Maybe make a linux .sh or something or anything that works with steam proton and linux? Idk.

**Comment by Breelandthecat** [comment]
Nvm. But i would still appreciate an official linux version.

**Comment by TheSillyDoggo** [comment]
there isn't even official linux gd you have to use proton

**Comment by Breelandthecat** [comment]
Yeah but it can still access the files. But the exe has to be ran in proton, but it doesn't work unless you add advanced launch options. And I think maybe making an sh file that either modifies the steam launch options for it, or something else to make it simpler for other people.
